Source/Editor/SceneGraph/Actors/StaticModelNode.cs

```diff
@@ -89,71 +89,87 @@ public override void OnContextMenu(ContextMenu contextMenu, EditorWindow window)
 
         private void OnAddMeshCollider()
         {
-            var model = ((StaticModel)Actor).Model;
-            if (!model)
-                return;
-
-            // Special case for in-built Editor models that can use analytical collision
-            var modelPath = model.Path;
-            if (modelPath.EndsWith("/Primitives/Cube.flax", StringComparison.Ordinal))
+            // Allow collider to be added to evey static model selection
+            var selection = Editor.Instance.SceneEditing.Selection.ToArray();
```

Comment:
Can you pass `EditorWindow window` from `OnContextMenu` here and handle using it in Prefab window?

Reply:
yes.